### PR TITLE
Replace the term "sanity", which is non-inclusive.

### PR DESCRIPTION
**Pull request by brycelelbach**

### CODE REVIEW

File: examples/CMakeLists.txt [hunk 1]
```diff
@@ -28,8 +28,8 @@ if (THRUST_ENABLE_EXAMPLE_FILECHECK)
   endif()
 
   execute_process(
-    COMMAND "${THRUST_FILECHECK_EXECUTABLE}" "${filecheck_data_path}/thrust.sanity.filecheck"
```
Comment (allisonvacanti) [inline examples/CMakeLists.txt:31]: Did this configure successfully when you tested it? I believe that this file in `internal/test/` will need to be updated too or this check will start failing. `git grep` shows a few other usages in that directory, too.

**Comment by tomilov** [comment]
What does it mean "non-inclusive"? Which property is not included into the meaning namely?

File: examples/CMakeLists.txt (hunk 1 above)
Comment (brycelelbach) [reply examples/CMakeLists.txt:31]: Fixed.

**Comment by brycelelbach** [comment]
> What does it mean "non-inclusive"? Which property is not included into the meaning namely?

The term "sanity", which has denotative (e.g. primary) meaning of "basic tests", also has negative connotations (e.g. secondary meanings) relating to mental health. The use of the term "sanity check" may suggest that individuals with mental illnesses are inferior, wrong, or incorrect.

In accordance with NVIDIA's code of conduct, we should strive to avoid non-inclusive terms like "sanity" and use neutral and inclusive language instead.

The term "sanity" is listed as a non-inclusive term to avoid by a number of technical and standards organizations:

* W3C: https://w3c.github.io/manual-of-style/#inclusive
* IETF: https://github.com/ietf/terminology
* Google Engineering: https://developers.google.com/style/inclusive-documentation
* Twitter Engineering
* INCITS (US national standards)
* ISO (international standards)

**Comment by brycelelbach** [comment]
run tests

**Pull request merged**